doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/gpu.rst

```diff
@@ -0,0 +1,200 @@
+    ray.get([actor.say_hello.remote() for actor in gpu_actors])
+
+After the program exits, the actors will be garbage collected.
+The GPU worker pods will be scaled down after the idle timeout (60 seconds by default).
+If the GPU worker pods were running on an autoscaling pool of Kubernetes nodes, the Kubernetes
+nodes will be scaled down as well.
+
+Requesting GPUs
+~~~~~~~~~~~~~~~
+You can also make a :ref:`direct request to the autoscaler<ref-autoscaler-sdk-request-resources>` to scale up GPU resources.
+
+.. code-block:: python
+
+    import ray
+
+    ray.init()
+    ray.autoscaler.sdk.request_resources(bundles=[{"GPU": 1} * 2])
+
+After the nodes are scaled up, they will persist until the request is explicitly overridden.
+The following program will remove the resource request.
+
+.. code-block:: python
+
+    import ray
+
+    ray.init()
+    ray.autoscaler.sdk.request_resources(bundles=[])
+
+The GPU workers can then scale down.
+
+Overriding Ray GPU capacity (advanced)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For specialized use-cases, it is possible to override the Ray pod GPU capacities advertised to Ray.
+To do so, set a value for the `num-gpus` key of the head or worker group's `rayStartParams`.
+For example,
+
+.. code-block:: yaml
+
+    rayStartParams:
+        # Note that all rayStartParam values must be supplied as strings.
+        num-gpus: "2"
+
+The Ray scheduler and autoscaler will then account 2 units of GPU capacity for each
+Ray pod in the group, even if the container limits do not indicate the presence of GPU.
+
+GPU pod scheduling (advanced)
+_____________________________
+
+GPU taints and tolerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note::
+
+  Managed Kubernetes services typically take care of GPU-related taints and tolerations
+  for you. If you are using a managed Kubernetes service, you might not need to worry
+  about this section.
+
+The `Nvidia gpu plugin`_ for Kubernetes applies `taints`_ to GPU nodes; these taints prevent non-GPU pods from being scheduled on GPU nodes.
+Managed Kubernetes services like GKE, EKS, and AKS automatically apply matching `tolerations`_
+to pods requesting GPU resources. Tolerations are applied by means of Kubernetes's `ExtendedResourceToleration`_ `admission controller`_.
+If this admission controller is not enabled for your Kubernetes cluster, you may need to manually add a GPU toleration each of to your GPU pod configurations. For example,
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+   generateName: example-cluster-ray-worker
+   spec:
+   ...
+   tolerations:
+   - effect: NoSchedule
+     key: nvidia.com/gpu
+     operator: Exists
+   ...
+   containers:
+   - name: ray-node
+     image: rayproject/ray:nightly-gpu
+     ...
+
+Node selectors and node labels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To ensure Ray pods are bound to Kubernetes nodes satisfying specific
+conditions (such as presence of GPU hardware), you may wish to use
```

Comment:
nit: There's a missing article: `s/such as presence of GPU hardware/such as the presence of GPU hardware/g`

Reply:
Thanks, will add the article :)

I wonder what the actual syntax rules here are
`Specific metaphysical conditions (such as existence)` but
`Specific conditions of your software system (such as _the_ presence of GPU hardware)`

Reply:
Article added.